docs/DeveloperGuide.md

```diff
@@ -12,11 +12,12 @@
     * [2.1. Project Overview](#21-project-overview)
         * [2.2. Module Overview](#22-module-overview)
```

Comment:
[2.2. Module Overview](#22-module-overview) should this section be outside of 2.1 instead of being placed under section 2.1?